classes/test/adapter/calls.php

```diff
@@ -320,6 +320,10 @@ protected function setCall(adapter\call $call, $position = null)
 			throw new exceptions\logic\invalidArgument('Function is undefined');
 		}
 
+		if (!isset(self::$callsNumber)) {
```

Comment:
It is, or not. If you are testing a resource that of kind `atoum://` stream, you will have an error. See https://github.com/Hywan/atoum/commit/d898c6d6111e98a38555a6ba93b2513198aa3007 for more “explanations”.